proposals/secure-storage.md

```diff
@@ -100,6 +100,8 @@ browser.secureStorage.store({
 });
 ```
 
+The authentication array is optional. If omitted, the secret is available without biometrics but is still stored in the hardware backed location.
```

Comment:
Instead of "without biometrics" I think it would be more accurate to say without "a recognized auth method"

Reply:
I've tweaked the wording here a little, let me know what you think. I also opened an issue just now that may impact how this works going forward: https://github.com/w3c/webextensions/issues/190